src/main/java/duke/util/List.java

```diff
@@ -0,0 +1,149 @@
+package duke.util;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+public class List {
```

Comment:
Maybe you can consider giving this class a more specific name, such as TaskList? I feel that List might be too general for this class as this class is specialised for tasklist only.